pkg/provision/providers/qemu/create.go

```diff
@@ -27,7 +27,7 @@ func (p *provisioner) Create(ctx context.Context, request provision.ClusterReque
 	}
 
 	arch := Arch(options.TargetArch)
-	if !arch.Valid() {
+	if err := arch.Valid(); err != nil {
```

Comment:
the returned error message is not used at all, so why change the signature to return an error

Reply:
Ah nice catch. I just forgot to pass the error here 🤦